components/Headlines.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import { useChannel } from "@ably-labs/react-hooks";
+import styles from "../styles/Home.module.css";
+import Image from "next/image";
+import { formatDate } from "../lib/dates";
+
```

Comment:
It feels like this file needs to be broken up further, or renamed - this isn't just "headlines"